python/mathsys/dev/__init__.py

```
@@ -61,6 +60,11 @@ def validate(content: str) -> bool:
     try: _parser.run(content); return True
     except: return False
 
+#> MAIN -> TOKENS
+@lru_cache(maxsize = None)
```

Ruff (0.14.6)

Use `@functools.cache` instead of `@functools.lru_cache(maxsize=None)`

Rewrite with `@functools.cache

(UP033)